nginx/nginx-router-mesh.conf.j2

```diff
@@ -81,7 +81,7 @@ http {
 			#proxy_cache_min_uses 3;
 			proxy_cache_valid      200  30s;
 			proxy_cache_use_stale  error timeout invalid_header updating http_500 http_502 http_503 http_504;
-            proxy_pass http://router-mesh/user-manager/v1/users;
+            proxy_pass http://router_mesh/user-manager/v1/users;
```

Comment:
My thinking is that {{ router-mesh.host }} holds the reference to the actual router-mesh service. By hardcoding "router_mesh" here, it ensures that this request is routed through the upstream rather than just getting sent directly to the service itself.

Reply:
Disregard my last note. I defer to @cstetson as he designed the MRA with the intention of running within the three NGINX Network Architectures